src/terminal7.ts

```diff
@@ -804,20 +798,37 @@ export class Terminal7 {
         })
     }
     toggleHelp() {
-        // TODO: add help for home & copy-mode
-        // var helpId = (this.activeG)? "help-gate":"help-home",
-        // var helpId = (this.activeG && this.activeG.activeW.activeP.copyMode)?
-        // "help-copymode":"help-gate",
-        const helpId = "help-gate",
-            ecl = document.getElementById(helpId).classList,
-            bcl = document.getElementById("help-button").classList
-            
-        ecl.toggle("show")
-        bcl.toggle("on")
-        if (!ecl.contains("show"))
-            this.focus()
-        // TODO: When at home remove the "on" from the home butto
+        if (!this.buttonToolTips)
+            this.createToolTips()
+        this.buttonToolTips.forEach(btt => {
+            btt.classList.toggle('hidden')
+        })
+        document.getElementById('keys-help').classList.toggle('hidden')
+        this.activeG?.activeW?.activeP?.hideSearch()
+    }
+
+    private buttonToolTips: HTMLDivElement[];
+
+    private createToolTips() {
+        const buttons = document.querySelectorAll('button[aria-label]')
+        const tooltips: HTMLDivElement[] = [];
+        buttons.forEach((b: HTMLButtonElement) => {
+            const tt = document.createElement('div')
+            if (b.dataset.sc) {
+                const shortCut = document.createElement('div')
+                shortCut.innerHTML = b.dataset.sc
```

Comment:
We've talked about it but I guess I wasn't clear enough. All our shortcuts are made from the Meta key & the shortcut. On Apples the meta is the Command key and on other it's ctrl-A. It's better to save just the key itself on the element and add the Meta here based on the platform.